test/run-spec-wasm2c.py

```diff
@@ -324,9 +367,14 @@ def _Action(self, command):
         mangled_module_name = self.GetModulePrefix(action.get('module'))
         field = mangled_module_name + MangleName(action['field'])
         if type_ == 'invoke':
-            return '%s(%s)' % (field, self._ConstantList(action.get('args', [])))
+            args = self._ConstantList(action.get('args', []))
+            if len(args) == 0:
+                args = '&' + mangled_module_name + "_module_instance"
+            else:
+                args = '&' + mangled_module_name + "_module_instance" + ', ' + args
```

Comment:
Using single quotes for strings. 

We tend to avoid string addition like this and prefer `'&%s_module_instance, %s' % (mangled_module_name, args)`... but maybe its time to switch to fstrings and do `f'&{mangled_name}_module_instance, {args}'`?

Reply:
Done.